doc/tutorials/biicode/tutorial_biicode/tutorial_biicode.markdown

```diff
@@ -0,0 +1,121 @@
+Using OpenCV with biicode dependency manager
+========================================
+
```

Comment:
falta un apartado Goal? 

- Easy install OpenCV for any OS
- How to detect faces in images using the [*objdetect module* from OpenCV]